src/main/java/org/shaolinmasters/akkadianlexicon/models/WebContents.java

```diff
@@ -0,0 +1,27 @@
+package org.shaolinmasters.akkadianlexicon.models;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Table(name = "web_contents")
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class WebContents {
```

Comment:
Az entitasok nevei mindig egyesszamban vannak, lsd King, Word, Source, stb. Itt is celszeru atirni WebContent-re.
A @Table annotacioban is akkor at kell irni egyes szamra. Ez az annotacio ugye elhagyhato lenne, mert a hibernate a camelcase-t snakecase-e alakitja automatikusan elvileg, gondolom csak az olvashatosag miatt raktad ki, ami jo otlet! ;)